lib/swig/structs.i

```diff
@@ -0,0 +1,252 @@
+typedef struct{
+} encrypt__Sha256Xor;
+
+typedef struct{
+    GoInt_ N;
+    GoInt_ R;
+    GoInt_ P;
+    GoInt_ KeyLen;
+} encrypt__ScryptChacha20poly1305;
+
+typedef struct{
+    GoUint32_  n[10];
+} secp256k1go__Field;
+
+
+typedef GoUint8_ cipher__PubKey[33];
+
+typedef GoUint8_ cipher__Ripemd160[20];
+
+typedef GoUint8_ cipher__SecKey[32];
+
+typedef GoUint8_ cipher__Sig[65];
+
+typedef GoUint8_ cipher__SHA256[32];
+
+typedef GoUint8_  cipher__Checksum[4];
+
+typedef struct{
+	GoUint8 data[33];
+} cipher_PubKey;
+
+typedef struct{
+	GoUint8 data[32];
+} cipher_SecKey;
+
+typedef struct{
+	GoUint8 data[20];
+} cipher_Ripemd160;
+
+typedef struct{
+	GoUint8 data[65];
+} cipher_Sig;
+
+typedef struct{
+	GoUint8 data[32];
+} cipher_SHA256;
+
+typedef struct{
+	GoUint8 data[4];
+} cipher_Checksum;
+
+typedef struct{
+	cipher_SecKey data[MAX_ARRAY_LENGTH_WRAP];
+	int count;
+} cipher_SecKeys;
+
+typedef struct{
+	cipher_PubKey data[MAX_ARRAY_LENGTH_WRAP];
+	int count;
+} cipher_PubKeys;
```

Comment:
Do not duplicate these . Use [SWIG `%include` directive](http://www.swig.org/Doc1.3/Preprocessor.html#Preprocessor_nn2) .

Put definitions above in `skycoin.cipher.crypto.i`